ct-app/core/api/hoprd_api.py

```
@@ -40,19 +50,22 @@
             headers = {"Content-Type": "application/json"}
             async with aiohttp.ClientSession(headers=self.headers) as s:
                 async with getattr(s, method.value)(
                     url=f"{self.host}{self.prefix}{endpoint}",
-                    json=data,
+                    json={} if data is None else data.as_dict,
                     headers=headers,
                 ) as res:
-                    return res.status, await res.json()
+                    try:
+                        return res.status, await res.json()
+                    except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)